R/SelfControlledCohort.R

```diff
@@ -223,6 +223,9 @@ runSelfControlledCohort <- function(connectionDetails,
     conn <- connectionDetails$conn
   }
 
+  DatabaseConnector::insertTable(conn, "tempdb..#scc_outcome_ids", data.frame(outcome_id = outcomeIds), tempTable = TRUE)
```

Comment:
This will not port across database platforms. I recommend using `tableName = "#scc_outcome_ids"`.

Also note that our code style guide [recommends using named arguments](https://ohdsi.github.io/Hades/codeStyle.html#Use_named_arguments). It is very easy to make a mistake if you don't!